### PR TITLE
fix: error in table dip_report_suivi_diplomation

### PR DESCRIPTION
**Pull request by girouardg**
# Objectives of the Pull Request ? 
> Corriger une erreur dans la requête SQL qui crée la table dip_report_suivi_diplomation.

# What is left out of the Pull Request ? 
> 

# How to run the pull request ?
> 

```bash
# Assuming you are in a folder containing both the core.data.store and the <cssXX>.data.store folder.
# Update the code
cd core.dashboards_store
git checkout bugfix/diplome
git pull
# Might be required if you update either the poetry file or the lock file
# poetry shell && poetry lock && poetry install 
cd ../csssh.dashboards_store
git checkout bugfix/diplome
git pull  
# Might be required if you add a new DBT dependency
# dbt deps
# Run dbt 
dbt build --select +tag:diplome+
```

# Checklist
> Please, read carefully each item before checking it. Your PR's review might be delayed otherwise.

* **Code** :
  * [x] The code I'm asking a review for is working. **I understand that my PR will be rejected as-is otherwise.**
  * [x] My code follows the conventions described in the `contribution guide`.
  * [ ] My `models` / `seeds` / `adapters` are tested and documented in DBT 9through the `yaml` files.
  * I have a added a new **mandatory seed** / a new **mandatory adapter** : 
    * [ ] I have you populated the `tooling/nightly` project with my new seed / adapter.
    * [ ] An exemple of how to populate the adapter can be found in the `analysis` folder.
  * I have added a new **dashboard** : 
    * [ ] I have updated the `core.dashboards_store/tooling/template/{{ cookiecutter.css_short_name + '.dashboards_store' }}/dbt_project.yml` file with my new dashboard. 
    * [ ] I have added a new `doc` page in the `core.dashboards_store/tooling/docs/content/` appropriate folder, describing what my dashboards does and how to configure it.
    * [ ] The dashboard is saved with `.pbit` extension, **not** a `.pbix` extension, so it doesn't contains any data.
  * [x] I have called `sqfmt .` to format my code, and **made sure my code was still running as expected after having been reformated**.
* **Documentation** : 
  * [ ]  I have updated the documentation (README) accordingly to my changes.
  * [ ]  The models I have added are documented in a `schema.yml` file.
* **Pull-request** : 
  * [x]  My merge message is prefixed with a `commitizen` key word so it looks something like `<feat|fix|chore|doc|refactor|perf|style>: foo bar`**
  * [x]  The code *provided to run the pull request* is working. **I understand that my PR will be rejected as-is otherwise.**
  * [ ]  My pull request is documented. I have explained the needs for the PR, what was left out of the it and why.
  * [x]  I have carefully reviewd each changes made to a file and made sure the files included on the PR were actually added on purpose.
  * [x]  I have assignee at least **1** people to review my PR.

### CODE REVIEW

**Review by jalluard — APPROVED** [review]

File: models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql [hunk 1]
```diff
@@ -367,7 +368,10 @@ with
             ) as nb_unites_g4_en_cours,  -- La somme des unités en cours en secondaire 4. Contient toutes les matières.
```
Comment (busquef) [inline models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql:368]: J'ai de la misère à comprendre le en_cours = 0 lorsque le nom de la variable spécifie en_cours.
On serait pas mieux de changer le nom ou d'utiliser en cours lorsque en_cours = 1?

**Comment by busquef** [comment]
Le tout me semble bon à part de mon autre commentaire

File: models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql (hunk 1 above)
Comment (sadqim) [reply models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql:368]: il faut développer cette partie... ca va être plus claire pour toi
image

**Review by sadqim — APPROVED** [review]
LGTM!

File: models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql (hunk 1 above)
Comment (busquef) [reply models/dashboards/diplome/pbi_tables/dip_report_suivi_diplomation.sql:368]: Je savais même pas qu'on pouvais extends. 

C'est bon de mon côté.

**Pull request merged**